doc/sphinx/addendum/ring.rst

```diff
@@ -694,6 +696,14 @@ The syntax for adding a new field is
       which is the completeness of equality on coefficients
       w.r.t. the field equality.
 
+   When :cmd:`Add Field` is called, a call to :cmd:`Add Ring` is performed with
+   all modifiers of the form :n:`@ring_mod`.  As a result, any previous ring
+   declaration for the type is replaced by the one that uses the same modifiers
+   as the :cmd:`Add Field` command.  In the case where it
+   is desired to have different modifiers for the field and the ring structure,
+   a new call to :cmd:`Add Ring` can be performed after this command, to set
+   different values of certain modifiers.
```

Comment:
("Suggest" is not working for me today)
How about this?

```
   :cmd:`Add Field` calls :cmd:`Add Ring` with
   its :n:`@ring_mod`\s.  This replaces any previous ring
   declaration for the type.
   To use different modifiers, use :cmd:`Add Ring` again after the
   :cmd:`Add Field`.
```

Seems like this should appear under `Add Field` rather than here in `Add Ring`.  Or in both if you prefer.

Reply:
@gares are you implementing this change?

Reply:
ping @gares